docs/vhost-user-blk-testing.md

```diff
@@ -0,0 +1,102 @@
+# How to test Vhost-user block with SPDK
```

Comment:
Inconsistent with below where you say "vhost-user-blk"

Reply:
Rob, thanks, i will change this.